src/index.ts

```diff
@@ -122,7 +145,7 @@ export default Tina;
 export { COND } from "./lib/conditions";
 
 /** Export EventEmitter Library */
-export { EventEmitter } from "./lib/events";
+export { EventEmitter } from "./lib/events/events";
```

Comment:
rename `events.ts` to `index.ts` and you'll be able to `"./lib/events";`

Reply:
Done.